.github/workflows/coverage.yml

```diff
@@ -48,10 +48,53 @@ jobs:
         env:
           COVERAGE_COVERAGE: "yes"
         run: |
-          python -m tox
+          set -xe
+          python -m tox -- -k fo
           python -m igor combine_html
+          mv .metacov .coverage-${{ matrix.python-version }}
+
+      - name: "Upload coverage data"
+        uses: actions/upload-artifact@v2
+        with:
+          name: metacov
+          path: .coverage-*
+
+  combine:
+    name: "Combine coverage data"
+    needs: coverage
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: "Check out the repo"
+        uses: "actions/checkout@v2"
 
-      - name: "Upload to codecov"
-        uses: codecov/codecov-action@v1
+      - name: "Set up Python"
+        uses: "actions/setup-python@v2"
         with:
-          file: coverage.xml
+          python-version: "3.9"
+
+      - name: "Install dependencies"
```

Comment:
Is there a way to make a common "action" of sorts?

Reply:
I was wondering about that.  There's a lot of repetition.